### PR TITLE
feature: plugin to use "imaginAIry" web api for generating images locally

### PR DESCRIPTION
**Pull request by brycedrennan**
Works with soon to be released version of [imaginAIry](https://github.com/brycedrennan/imaginAIry).

`getStableDiffusionDefaultCount` and `getStableDiffusionDefaultCount` appear to be ignored by the ui

### CODE REVIEW

**Comment by hlky** [comment]
> `getStableDiffusionDefaultCount` appear to be ignored by the ui

Can confirm, found the same with ComfyUI plugin, [fix](https://github.com/Stability-AI/StableStudio/pull/36/files#diff-759ef80471a749ceee594b416a9d5958225baf0cb6484946e5381f5882e16d6f) is included in that PR

**Comment by brycedrennan** [comment]
nice - thanks!

**Comment by cruhl** [comment]
@brycedrennan What are your thoughts on this comment? ~> https://github.com/Stability-AI/StableStudio/pull/36#issuecomment-1559660704

TL;DR – We're thinking it's time to make the DX around third-party plugins better so folks aren't bottlenecked by us here.

**Comment by cruhl** [comment]
> 

Regardless of what we decide, I'm excited to help out with this 👍

**Comment by brycedrennan** [comment]
@cruhl seems like a good strategy on a lot of levels

**Review by KAJdev — COMMENTED** [review]
:eyes:
